modules/GKB/EnsEMBLMartUtils.pm

```diff
@@ -48,44 +50,46 @@ our %EXPORT_TAGS = (all => [@EXPORT_OK],
                    query => [qw/get_query get_query_runner/]);
 
 sub get_species_results {
-    my $species = shift;
-    return get_species_results_with_attribute_info($species)->{'results'};
+    my $logger = get_logger(__PACKAGE__);
+
+    my $species_abbreviation = shift // $logger->logconfess("Need species abbreviation");

```

Comment:
So what will the value of `$species_abbreviation` be if the other side of `//` executes? I imagine it will be NULL. Is it OK to pass null into `get_species_results_with_attribute_info`?

Reply:
Apparently, `logconfess` _will_ break execution, in addition to logging a message.